src/widgets/helper/CommonTexts.hpp

```diff
@@ -1,8 +1,13 @@
 #pragma once
 
-#define OPEN_IN_BROWSER "Open stream in browser"
-#define OPEN_PLAYER_IN_BROWSER "Open player in browser"
-#define OPEN_MOD_VIEW_IN_BROWSER "Open mod view in browser"
-#define OPEN_IN_STREAMLINK "Open in streamlink"
-#define DONT_OPEN "Don't open"
-#define OPEN_WHISPERS_IN_BROWSER "Open whispers in browser"
+#include <QString>
+
+inline const QString OPEN_IN_BROWSER = QStringLiteral("Open stream in browser");
```

Comment:
do we avoid using the `u"..."_s` syntax here to keep the header file simpler?

Reply:
We can't use it in header files, as we'd need to add a `using` directive - effectively making the operator `""_s`  available everywhere.